src/modules/interfaces.ts

```diff
@@ -147,6 +147,11 @@ export class Crud<E> {
           logger.info(`Cache miss for ${this.entity?.name ?? ''} ${this.dest}`);
           ctx.memo[dest][entityName][id] = new (this.entity as new () => E)();
         } else {
+          // If object is empty it means it is a placeholder and it is not in the memo yet.
+          if (!Object.keys(ctx.memo[dest][entityName][id]).length) {
+            logger.info(`Cache miss for ${this.entity?.name ?? ''} ${this.dest}`);
+            return undefined;
+          }
```

Comment:
If this is staying in, should we just drop the placeholder logic? They aren't "getting through" to the users so they're just excessive complexity at this point?

Reply:
I'll create a follow up task with this and try to tackle in a new PR.